src/components/subject-level-with-labels/SubjectLevelWithLabels.styles.ts

```diff
@@ -0,0 +1,24 @@
+import { TypographyVariantEnum } from '~/types'
+
+export const styles = {
+  container: {
+    display: 'flex',
+    gap: '10px'
+  },
+  labels: {
+    display: 'flex',
+    flexDirection: 'column',
+    gap: '8px'
+  },
+  label: {
+    typography: TypographyVariantEnum.Overline,
+    fontSize: '10px',
```

Comment:
![image](https://github.com/ita-social-projects/SpaceToStudy-Client/assets/97129041/39f0a50b-58e3-463b-a6ca-fde214f6de8e)
overline already has fontsize,weight,transform

Reply:
removed that